Support `capsule build --release --debug-info`

Flouse opened the pull request:
Use `--debug-info` arg to generate full debug info.

## Example
$ godwoken-scripts >
capsule build --release --debug-output --debug-info

Flouse committed:
Support `capsule build --release --debug-info`

Use `--debug-info` arg to generate full debug info.

## Example
$ godwoken-scripts >
capsule build --release --debug-output --debug-info

xxuejie commented on src/recipe/rust.rs:
```diff
@@ -57,14 +57,20 @@ impl Rust {
+                    args.push_str("-g ");
                 } else {
-                    format!("RUSTFLAGS=\"{} {}\"", BASE_RUSTFLAGS, RELEASE_RUSTFLAGS)
+                    args.push_str(&format!("{} ", RELEASE_RUSTFLAGS));
```
`RELEASE_RUSTFLAGS` is a very generic name, chances are more flags for release version might be included here. What we should do, is to introduce a new flag just for handling ELF stripping. When `debug_info` is enabled, only the flag for ELF stripping should be removed, there might well be other release version RUSTFLAGS that shall be used even when `debug_info` is enabled.

Flouse committed:
Introduce STRIP_RUSTFLAG handling binary stripping

move stripping option out of RELEASE_RUSTFLAGS

> https://github.com/nervosnetwork/capsule/pull/37/files#r690004395

TheWaWaR commented:
Fixed in #47
